datalad_registry/utils/__init__.py

```diff
@@ -0,0 +1 @@
+from .misc import StrEnum  # noqa: F401(imported but unused)
```

Comment:
not clear why bother importing here then at all

Reply:
I was thinking making `StrEnum` immediately available under the `utils` package.